blog/README.md

```diff
@@ -5,15 +5,14 @@ The Knative website has a basic, top-level navigation that looks like this:
 ```yaml
 nav:
     - Home:
-    - Getting started:
+    - Tutorial:
     - Installing:
     - Serving:
     - Eventing:
     - Code samples:
     - Reference:
-    - About:
     - Blog:
-    - Case studies:
+    - About:
     - Community:
```

Comment:
Currently the order in this snippet matches the order we can see on the site (Blog, About, Community), but should this match the order in the `nav.yml` (Community, About, Blog)?

Reply:
For some reason the last items are reversed on the site from what is in nav.yaml

I will update to match nav.yaml